packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -476,6 +476,22 @@
       <label>Copié!</label>
       <name>quantic_Copied</name>
     </customLabels>
+    <customLabels>
+      <label>Oui</label>
+      <name>quantic_Yes</name>
+    </customLabels>
+    <customLabels>
+      <label>Non</label>
+      <name>quantic_No</name>
+    </customLabels>
+    <customLabels>
+      <label>Est-ce que cela a été utile?</label>
+      <name>quantic_WasThisUseful</name>
+    </customLabels>
+    <customLabels>
+      <label>Expliquer pourquoi</label>
```

Comment:
Very minor comment here. Just wondering do we wanna say Expliquer or Expliquez? Not sure which one would make more sense for feedback since you are addressing the user directly. Up to you, mostly food for thought :P

Reply:
Very good point!
What do you think @acarpentier01 , `Expliquer pourquoi` or `Expliquez pourquoi` ?

Reply:
"Expliquez pourquoi" gives it more of a personal touch. That said, I think I'd go with "Dites-nous pourquoi".

Reply:
Thank you!